app/views/spree/admin/products/seo.html.haml

```diff
@@ -2,7 +2,8 @@
 = render :partial => 'spree/admin/shared/product_tabs', :locals => { :current => t(:Search) }
 = render :partial => 'spree/shared/error_messages', :locals => { :target => @product }
 
-= form_for [:admin, @product], :method => :put, :html => { :multipart => true } do |f|
-  %fieldset.no-border-top
-    = render :partial => 'seo_form', :locals => { :f => f }
-    = render :partial => 'spree/admin/shared/edit_resource_links'
+%div{ 'ng-app' => 'ofn.admin' }
```

Comment:
Why did you add the app? What that necessary to use `ofn-with-tip`?

Reply:
I'm no Angular specialist but it seems to be the way to link this view to the controller so that the tip is triggered.